test/utils.js

```diff
@@ -36,11 +36,26 @@ export function almostEqual(a, b, criteria) {
   }
 }
 
-export function checkValue(
-    output, expected, criteria = opFp32AccuracyCriteria) {
+function getBitwise(value) {
+  const buffer = new ArrayBuffer(4);
+  const view = new DataView(buffer);
```

Comment:
as I commented before, you should populate the buffer https://github.com/webmachinelearning/webnn-polyfill/pull/139#discussion_r746318307